### PR TITLE
Species chameleon fix

### PR DESCRIPTION
**Pull request by Markolie**
**What does this PR do:**
This pull request ensures chameleon items use the appropriate species icons. It also moves the chameleon file to a more appropriate folder and removed the unused species_fit variable.

**Changelog:**
:cl: Markolie
fix: Chameleon items now use the appropriate species icon.
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/chameleon.dm [hunk 1]
```diff
@@ -323,6 +351,16 @@
 	permeability_coefficient = 0.01
 	flags_cover = MASKCOVERSEYES | MASKCOVERSMOUTH
 
+	species_fit = list("Vox", "Unathi", "Tajaran", "Vulpkanin", "Grey")
```
Comment (KasparoVy) [inline code/modules/clothing/under/chameleon.dm]: Drask is missing from this list but is present under `sprite_sheets`

File: code/modules/clothing/under/chameleon.dm [hunk 2]
```diff
@@ -238,6 +254,13 @@
 	resistance_flags = NONE
 	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
 
+	species_fit = list("Vox")
```
Comment (KasparoVy) [inline code/modules/clothing/under/chameleon.dm]: `drask` and `grey` may need to be in this list as well afaik

Comment (Markolie) [reply code/modules/clothing/under/chameleon.dm]: I just copied these from their parent (i.e. gas mask), so looks like those are incorrect there as well. On further inspection, I don't think species_fit is even used anywhere. I'm just going to delete the variable entirely.

Comment (KasparoVy) [reply code/modules/clothing/under/chameleon.dm]: Sweet. Yeah on second thought, looking at the procs in `clothing.dm` it seems to only really refer to `sprite_sheets` anyhow

**Pull request merged**